src/__tests__/ProductView/ConfigurationHeader.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ConfigurationHeader from '../../ProductView/Header/ConfigurationHeader/ConfigurationHeader.js';
+import renderer from 'react-test-renderer';
+import TestProduct from './../testData/product.js'
+
+let instance, component;
+
+const firstTestConfiguration = TestProduct.PRODUCT.configurations[0];
+const secondTestConfiguration = TestProduct.PRODUCT.configurations[1];
+const setProductConfiguration = jest.fn();
+
+test("Rendering of ConfigurationHeader", () => {
+    component = renderer.create(
```

Comment:
extract this to a beforeAll method. Example __tests__/ProductView/QueryInterface.js.